logger.gd

```diff
@@ -103,10 +116,7 @@ class Logfile:
 				flush_buffer()
 
 	func get_config():
-		return {
-			"path": get_path(),
-			"queue_mode": get_queue_mode()
-		}
+		return {"path": get_path(), "queue_mode": get_queue_mode()}
```

Comment:
Does it keep them split if you add a trailing comma?

Reply:
Yes it does!

Reply:
I'd say it would be good to do then, it's more readable IMO.